p_steps/step_12_1_create_scri_dataset.R

```diff
@@ -0,0 +1,293 @@
+# Program Information  ----------------------------------------------------
+
+# Program:      step_12_1_create_scri_dataset.R 
+# Author:       Anna Schultze; Ema Alsina, Sophie Bots, Ivonne Martens 
+# Description:  calls a function which creates SCRI dataset in wide format 
+#               takes user provided brands and outcomes
+#               outputs datasets provided N is >5. 
+#               if N is < 5 no output data is provided and a warning printed to console 
+# Requirements: 
+#               dependencies: 07_scri_inputs.R
+#               input:  D3_study_variables_for_SCRI (g_intermediate) in RData format 
+#               parameters: in 07_scri_inputs.R
+#               output: g_output/scri/scri_input_[brand]_[outcome]_flowchart.txt
+#                       g_intermediate/scri/scri_input_[brand]_[outcome]_input.csv               
+
+# Housekeeping  -----------------------------------------------------------
+# install and load packages 
+if (!require("pacman")) install.packages("pacman")
+pacman::p_load(packages, character.only = TRUE)
+
```

Comment:
I just tried to run this on my computer, but I get the error that `gtsummary` failed to load.
This is the error I get:

_Error: package or namespace load failed for ‘gtsummary’:
 object ‘faststack’ is not exported by 'namespace:fastmap'_

In addition it tries to connect to the (I think) London School R package repository, which of course I cannot access. R solves this itself, but maybe we can change the code so that it automatically picks the local CRAN repository instead of the London one.